hackathon_site/registration/models.py

```diff
@@ -56,12 +56,30 @@ class Application(models.Model):
         ("other", "Other"),
     ]
 
+    TSHIRT_SIZE_CHOICES = [
+        ("small", "S"),
```

Comment:
Like with the other options, need to specify a blank option `(None, "")`, otherwise the template renders weird. Same for the one below